### DISCUSSION TITLE
Scaling to 170 million vectors

### DISCUSSION THREAD

**Discussion by Kencho5** [milvus-io/milvus#15492] [Q&A and General discussion]
hello,

i have around 170 million 512 dimensional vectors, i'm using **IVF_SQ8** index, nlist is 100. milvus sizing tool estimated around 82GB of ram needed to scale to this and loading the collection in memory takes quite some time even on 10 million vectors so i'm afraid on 170 million it may even take hours? when searching i set nprobe to 8. does anybody have any tips for me to scale to it better and get very fast search results?

my specs are: 

- CPU: i7 9700K
- RAM: 16GB (for now)
- GPU: RTX 3060 12gb

**Comment by wangting0128** [comment]
### The following configuration you can use as a reference

1. 100 million 128 dimensional vectors，index type is ivf_sq8 and nlist of index is 2048
resource settings for querynode：
```
queryNode:
  replicas: 1
  resources:
    limits:
      cpu: "8.0"
      memory: 64Gi
```
The time it takes to load the collection is: **436.2741s**

**Reply by Kencho5** [reply]
i see so loading does take quite some time. what about searching on that configuration?

**Reply by wangting0128** [reply]
**search time is about 1.9s**

the params of search as following：
```
top_k: 10
nq: 10
search_param:
    nprobe: 16
```

**Comment by yanliang567** [comment]
@Kencho5 it seems that your RAM is not enough for loading 170 million vectors:(

**Reply by Kencho5** [reply]
yes it’s not enough, but i’m just testing on small scale for now. i’ll upgrade when i’ll go to full scale
